core-java-modules/core-java-date-operations-3/src/test/java/com/baeldung/timeunitconversions/TimeUnitConversionsUnitTest.java

```diff
@@ -0,0 +1,90 @@
+package com.baeldung.timeunitconversions;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TimeUnitConversionsUnitTest {
+
+
+    @Test
+    public void givenTotalInput_whenConvert_thenCorrectTotalResult() {
```

Comment:
As this is JUnit 5, we don't need the `public` on the class or the test methods.